### PR TITLE
search and explore page related changes, bottom naviagtion routing issue fix

### PR DESCRIPTION
**Pull request by 5Amogh**
…

### CODE REVIEW

File: src/app/generic-listing-page/generic-listing-page.component.ts [hunk 1]
```diff
@@ -76,4 +96,11 @@ export class GenericListingPageComponent  implements OnInit {
     this.count = 0
   }
 
+  handleInput(event: any) {
+    this.searchTerm = event.target.value;
+    this.page = 1;
```
Comment (Vinod-V3) [inline src/app/generic-listing-page/generic-listing-page.component.ts:101]: the the below 3 line u can call reset() function

File: src/app/generic-listing-page/generic-listing-page.component.html [hunk 2]
```diff
@@ -1,10 +1,27 @@
-<ion-content>
+<app-application-header [config]="headerConfig" (actionClick)="handleActionClick($event)">
+<ng-template #search>
   @if(pageConfig.enableSearch){
-    <!-- To be replaced with search component -->
-    <div>SEARCH BAR</div>
+    <ion-row class="srch">
```
Comment (Vinod-V3) [inline src/app/generic-listing-page/generic-listing-page.component.html:4]: Can u rename the class to "search"

File: src/app/generic-listing-page/generic-listing-page.component.html [hunk 3]
```diff
@@ -1,10 +1,27 @@
...
+      <ion-col size="12">
+        <ion-searchbar placeholder="Hinted search text" [debounce]="1000"
```
Comment (Vinod-V3) [inline src/app/generic-listing-page/generic-listing-page.component.html:6]: placeholder text also should come from en.json

File: src/app/generic-listing-page/generic-listing-page.component.ts (hunk 1 above)
Comment (5Amogh) [reply src/app/generic-listing-page/generic-listing-page.component.ts]: done

File: src/app/config/listingPageConfig.ts [hunk 4]
```diff
@@ -2,7 +2,10 @@ export const listingConfig = {
     explore: {
         apiUrl: "/project/v1/library/categories/projects/educationLeader",
         enableSearch: true,
-        headerConfig: {}
+        headerConfig: {
+            title:'Explore',
```
[the remark this answers is not in the document]
Comment (5Amogh) [reply src/app/config/listingPageConfig.ts]: updated

File: src/app/generic-listing-page/generic-listing-page.component.html (hunk 2 above)
Comment (5Amogh) [reply src/app/generic-listing-page/generic-listing-page.component.html]: updated

File: src/app/generic-listing-page/generic-listing-page.component.html (hunk 3 above)
Comment (5Amogh) [reply src/app/generic-listing-page/generic-listing-page.component.html]: done

**Review by Vinod-V3 — APPROVED** [review]

**Pull request merged**